service/src/main/java/org/apache/griffin/core/metric/MetricStoreImpl.java

```diff
@@ -211,9 +213,13 @@ private static String buildBasicAuthString(String user, String password) {
 
     @Override
     public MetricValue getMetric(String applicationId) throws IOException {
-        Map<String, String> map = new HashMap<>();
-        map.put("q", "applicationId:" + applicationId);
-        Response response = client.performRequest("GET", urlGet, map);
+        Response response0 = client.performRequest(
```

Comment:
Looks like this value never used.

Reply:
@dershovGD thanks a lot, it was debug stuff. Let me get rid of it...